### PR TITLE
add assets that are missing, fix docs and oracles

### PR DESCRIPTION
**Pull request by carlomazzaferro**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

**Review by coolnft — APPROVED** [review]
changes looks good to me.

File: packages/sdk/src/chainConfig/assets/polygon.ts [hunk 1]
```diff
@@ -221,40 +221,54 @@ export const assets: SupportedAsset[] = [
     underlying: "0x2ffbce9099cbed86984286a54e5932414af4b717",
```
Comment (peetzweg) [inline packages/sdk/src/chainConfig/assets/polygon.ts:221]: This one is using the checksum address so mixed case version. Every other address seems to be just lowercased.

Comment (carlomazzaferro) [reply packages/sdk/src/chainConfig/assets/polygon.ts]: changed to `0x2fFbCE9099cBed86984286A54e5932414aF4B717` -- is this correct?

Comment (carlomazzaferro) [reply packages/sdk/src/chainConfig/assets/polygon.ts]: looks like it :)

**Pull request merged**